apps/web/components/eventtype/EventAdvancedTab.tsx

```diff
@@ -125,6 +127,7 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupProps,
         <TextField
           label={t("event_name_in_calendar")}
           type="text"
+          {...shouldLockDisableProps("eventName")}
```

Comment:
This function returns `disabled` and `isLocked` properties for a given field name, `TextField` now supports `isLocked` to be able to show an indicator in the label, indicator that comes in the `isLocked` flag itself.